site/docs/v1/tech/apis/identity-providers/_links-post-response-body.adoc

```diff
@@ -0,0 +1,17 @@
+==== Response Body
+
+[.api]
+[field]#identityProviderLink.identityProviderId# [type]#[UUID]#::
+The unique Id of the identity provider.
+
+[field]#identityProviderLink.identityProviderUserId# [type]#[String]#::
+The Id for the User that is provided by the identity provider.
+
+[field]#identityProviderLink.userId# [type]#[UUID]#::
+The FusionAuth User Id that is linked to the identity provider.
```

Comment:
I don't think this is complete. In the domain object I see `data`, `displayName`, `identityProviderId`, `identityProviderUserId`, `insertInstant`, `lastLoginInstant`, `tenantId`, `token` and `userId`. 

Is this a specialized response example?